Dockerfile.ubuntu

```diff
@@ -0,0 +1,4 @@
+FROM ubuntu
```

Comment:
Perhaps it would be more approriate to have this inside a `contrib/ubuntu/Dockerfile` instead. This is quite unrelated with the core of the project.